### ISSUE TITLE
`Communication`: Keep order of replies after marking solution

### ISSUE THREAD

**Issue by dfuchss** [ls1intum/Artemis#7089]
### Describe the solution you'd like

If I mark an answer as solution, the answer is moved quite at the top (below the questions). This makes it hard to follow the previous discussion. because it is not ordered anymore. I'd suggest that the message is not moved on marking.

**Comment by dfuchss** [comment]
@egekurt123 how do you handle this in the new Communication/Messaging module?

**Comment by krusche** [comment]
the order of messages or replies should not change when marking a reply as resolving the message
